extension-iap/manifests/android/AndroidManifest.xml

```diff
@@ -4,9 +4,9 @@
 <uses-sdk android:minSdkVersion="{{android.minimum_sdk_version}}" android:targetSdkVersion="{{android.target_sdk_version}}" />
     <application>
         <!-- For Amazon IAP -->
-        <receiver android:name="com.amazon.device.iap.ResponseReceiver" >
+        <receiver android:name = "com.amazon.device.iap.ResponseReceiver" android:permission = "com.amazon.inapp.purchasing.Permission.NOTIFY" >
```

Comment:
Looks good. Source: https://developer.amazon.com/docs/in-app-purchasing/iap-implement-iap.html#responsereceiver